src/theme-editor/src/ThemeEditor.tsx

```diff
@@ -53,6 +53,13 @@ const ColorWandIcon = (
   </svg>
 )
 
+// button colorOpacitySecondary var is number
+const judgeUseColorPicker = (key: string, value: number | string): boolean => {
```

Comment:
```suggestion
const WhetherToUseColorPicker = (key: string, value: number | string): boolean => {
```
或者

```suggestion
const isColorPicker = (key: string, value: number | string): boolean => {
```

这样命名会不会好一点

Reply:
应该会